examples/federation-workshop/cluster-1/bundle/Endpoint.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: spire-server-bundle-endpoint
+  namespace: spire-system
+spec:
+  type: LoadBalancer
```

Comment:
In a real-world deployment this can be an Ingress instead of a service of type LoadBalancer.

Also, to get external IP, I’m using MetalLB and for MetalLB to work with minimal issues I’m using microk8s (I had nasty routing issues with Minikube; and I haven’t tried it with kind). Again, in a real-world setup with cloud load balancers, or a bare metal k8s with metallb things will work fine.